addOns/wappalyzer/src/main/javahelp/org/zaproxy/zap/extension/wappalyzer/resources/help/contents/options.html

```diff
@@ -17,8 +17,9 @@ <H3>Mode</H3>
   <li>Exhaustive > Keep matching and don't return early; likely slightly less performant.</li>
 </ul>
 
-<H3>Alerts</H3>
+<H3 id="10004">Alerts</H3>
 Allows the user to select whether or not the technology detection passive scan rule should raise alerts as technologies are identified (Enabled by default).
+Alerts are Informational and include the Tech Name, a Description of the tech if available, Other Info may include CPE or version details, and References may contain Tech specific URLs.
```

Comment:
Could also link to the class like other scan rules do (even if not exactly the same type).